tools/ci_build/build.py

```diff
@@ -79,6 +79,7 @@ def _openvino_verify_device_type(device_read):
         "CPU_NO_PARTITION",
         "GPU_NO_PARTITION",
         "NPU_NO_PARTITION",
```

Comment:
Can we remove NPU_NO_PARTITION . It does not mean anything. We never do graph partitioning for NPU